src/components/interactive-image/interactive-image.scss

```diff
@@ -0,0 +1,33 @@
+@import "../common/_base";
+
+@mixin size($width, $height) {
+  height: $height;
+  width: $width;
+}
+
+gx-interactive-image {
+  /**
+  * @prop --control-height:
+   * Set the height or the control. *Must be set before render the control*
+   * (auto by default)
+  */
+  --control-height: false;
+
+  /**
+	* @prop --control-width:
+	 * Set the width or the control. *Must be set before render the control*
+	 * (100% by default)
+	*/
+  --control-width: false;
```

Comment:
Why do we need new custom CSS properties for setting the dimensions? Shouldn't we just use `width` and `height` like a regular HTML element?

Reply:
My bad. I forgot to return it to the original value. I'll fix that

Reply:
@ncardeli those css vars works for the img container. not for the img tag

Reply:
I understand, but why should I use those properties instead of directly using `width` and `height` on the custom element?

Reply:
If we just pass `width` and `height` as attributes... The component read it well? 😯

Reply:
I try it and it doesn't work. Do you say add it as properties?

Reply:
Values fixed in da6f90c

If is desirable to add `width` and `height` as properties of the control and be available to set the size using those properties, ask me to add it!

Reply:
I applied your recommendation!
Look at the commit 0044d2e